uniffi_bindgen/src/bindings/swift/templates/EnumTemplate.swift

```diff
@@ -16,11 +16,11 @@ public enum {{ e.name()|class_name_swift }}: Lowerable, Liftable {
         }
     }
 
-    func lower(into buf: Writer) {
-        self.toFFIValue().lower(into: buf)
+    func write(into buf: Writer) {
+        self.lower().write(into: buf)
```

Comment:
I'm not super familiar with visibility rules in Swift, will these `read` and `write` methods be visible to consumers by default? I'm wondering if we need to e.g. put them in an extension block in order to take them out of the public API.

Reply:
I think by default they won't

Reply:
By default they're `internal`, i.e. won't be accessible outside of the module.

[source][1]

[1]: https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html#//apple_ref/doc/uid/TP40014097-CH41-ID5